ngraph/core/include/openvino/core/node_output.hpp

```diff
@@ -77,6 +77,13 @@ class OPENVINO_API Output<Node> {
     /// \return The constant reference to runtime info map
     const RTMap& get_rt_info() const;
 
+    /// \return The tensor names associated with this output
+    const std::unordered_set<std::string>& get_names() const;
+    /// \return Set tensor names associated with this output
+    void set_names(const std::unordered_set<std::string>& names);
+    /// \return Add tensor names associated with this output
+    void add_names(const std::unordered_set<std::string>& names);
```

Comment:
will we have:
```c++
std::string get_name() const { return *get_names().begin(); }
```
?

Reply:
But if I need any name, it would be nice to have a short cut for this, IMO.

Reply:
See https://github.com/openvinotoolkit/openvino/pull/7829#discussion_r721353720 and ticket XXX-67353

Reply:
We still use get_name method. We shouldn't change the behavior of this method.